src/entity/user/experiences/UserExperience.ts

```diff
@@ -69,6 +69,13 @@ export class UserExperience {
   @Column({ type: 'text', default: null })
   locationId: string | null;
 
+  @Column({ type: 'jsonb', default: {} })
+  customLocation: Partial<{
```

Comment:
will expose this column and connect to frontend for readonly display on profile in different PR.

Reply:
Write should always go to `locationId`